README.md

```diff
@@ -107,11 +108,42 @@ the file in `vars/gpg-keys`.
 ### Logging into ArgoCD
 
 ```bash
-# Login to ArgoCD
 ARGOCD_ROUTE=$(oc get route argocd-server -o jsonpath='{.spec.host}')
+# Login to ArgoCD as admin user
 argocd --insecure --grpc-web login ${ARGOCD_ROUTE}:443 --username admin --password ${ARGOCD_SERVER_PASSWORD}
+# Login to ArgoCD via SSO
+argocd --insecure --grpc-web login ${ARGOCD_ROUTE}:443 --sso
+```
+
+## Adding Clusters and namespaces
+
+```bash
+argocd cluster add <cluster_context> --namespace <application_namespace>
 ```
 
+Note: You can find your cluster-contexts by going into your kubeconfig or by running argocd cluster add with no arguments.
+
+### Deploying Applications with ArgoCD
+
+1. Log into Argocd ui at ```echo https://$(oc get routes | grep argocd-server | awk 'NR==1{print $2}')```
+2. Click create application
+3. Fill the fields out like so:
+
+    Application name: <application_name>
+    Project: default
+    Sync Policy: Automatic (or manual)
+    Sync Option: Check Use A Schema...
+    Repository URL: <git_url>
+    Revision: <branch_name> or HEAD
+    Path: <path_to_overlay> or .
+    Cluster: Your cluster address
+    Namespace: <application_namespace>
+
+Leave the rest blank and click create at the top left.
+If everything works the creation slider will go away, refresh the page and the app should be there if isn't already. Click sync and your application should be deployed.
+
+NOTE: You may need to disable schema validation if your deployments are failing. This is due to a bug with the ArgoCD api validator.
```

Comment:
I don't think it's technically a bug - the api spec they use is very restrictive, so I'd leave that part out.